projects/kvm/src/lib/kvm.component.spec.ts

```diff
@@ -15,13 +17,81 @@ describe('KvmComponent', () => {
       .compileComponents()
   })
 
-  beforeEach(() => {
+  beforeEach(fakeAsync(() => {
     fixture = TestBed.createComponent(KvmComponent)
     component = fixture.componentInstance
+    component.mpsServer = 'wss://localhost'
```

Comment:
im not sure we should do this in the beforeEach -- the default state is that mpsServer and authToken are empty strings. So i think we should leave that alone, and change them later in the tests where we need it.

Reply:
updated